app/models/user.rb

```diff
@@ -89,9 +89,16 @@ class User < ApplicationRecord
   has_many   :playlists, -> { order('playlists.position') },
     dependent: :destroy
 
+  # this only covers comments received by the user
+  # not comments made by the user
   has_many   :comments, -> { order('comments.id DESC') },
     dependent: :destroy
 
+  has_many :given_comments, -> { order('comments.id DESC') },
```

Comment:
I believe elsewhere in the code/specs, this is named `comments_made` — I think "made" is clearer than "given" but we should stick with one name. We could consider renaming the `comments` scope to `comments_received` to help the naming become more clear.